README.md

```diff
@@ -29,26 +29,27 @@ Mina is the first cryptocurrency with a lightweight, constant-sized blockchain.
 
 ## Contributing
 
-Read the [Contributing Guide](https://docs.minaprotocol.com/node-developers/contributing) for information on how to make both technical and non-technical contributions.
+For information on how to make technical and non-technical contributions, see the repository contributing guidelines in [CONTRIBUTING](https://github.com/MinaProtocol/mina/blob/develop/CONTRIBUTING.md) and the [Contributing Guide](https://docs.minaprotocol.com/node-developers/contributing) docs.
 
 ## Developers
 
-The [Developer docs](https://docs.minaprotocol.com/node-developers) contain all the information needed to begin contributing code to Mina, and using Mina APIs to build applications.
+The [Node Developers](https://docs.minaprotocol.com/node-developers) docs contain helpful information about contributing code to Mina and using Mina APIs to build applications.
 
-#### Quick Links:
+### Quick Links
 
 - [Developer README](README-dev.md)
 - [Running a demo node](docs/demo.md)
 - [Lifecycle of a payment](https://docs.minaprotocol.com/node-operators/lifecycle-of-a-payment)
 
 ## Community
 
-- Join the public [Discord server](https://bit.ly/MinaDiscord)! Please come by if you need help or have any questions.
-- Get the latest updates by [signing up for the Mina newsletter](https://docs.google.com/forms/d/e/1FAIpQLSdChigoRhyZqg1RbaA6ODiqJ4q42cPpNbSH-koxXHjLwDeqDw/viewform?usp=pp_url&entry.2026041782=I+just+want+to+learn+more!).
+- Join the public Mina Protocol [Discord server](https://discord.gg/minaprotocol). Please come by if you need help or have any questions.
+- Participate in our [online communities](https://docs.minaprotocol.com/participate/online-communities).
+- Get the latest updates by signing up for the Mina newsletter. Select [SIGN UP FOR NEWSLETTER](https://minaprotocol.com/) on the home page of the Mina Protocol website.
 
 ## Follow our work
 
-We use ZenHub to track our tasks and get metrics about our team's efforts. ZenHub is a third-party product that adds an abstraction layer with more functionalities on top of GitHub. You can follow our work in the following boards:
+We use Zenhub to track our tasks and get metrics about our team efforts. The Zenhub third-party product adds an abstraction layer and adds functionalities on top of GitHub. You can follow our work in the following boards:
```

Comment:
Is this still true? In MF we tried using Zenhub for a brief moment, but quickly switched back to working directly with Github issues and PRs. Maybe O(1) uses Zenhub, though.

Reply:
I removed this section, iterative improvements in future can provide more context if required